src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -100,45 +100,21 @@ const MessagesVirtualizedList = (props: Props) => {
 		firstNextChunkIsLoaded,
 	]);
 
-	const debouncedScrollHandler = useDebouncedCallback(
-		(
-			event: React.UIEvent<'div'>,
-			isHorizontal?: boolean,
-			wheelScrollDirection?: 'next' | 'previous',
-		) => {
-			const scroller = event.target;
-			if (scroller instanceof Element) {
-				const isStartReached = scroller.scrollTop === 0;
-				const isEndReached = scroller.scrollHeight - scroller.scrollTop === scroller.clientHeight;
-				if (
-					isStartReached &&
-					searchChannelNext &&
-					!searchChannelNext.isLoading &&
-					!isHorizontal &&
-					!updateStore.isActive &&
-					((wheelScrollDirection === undefined &&
-						scroller.parentElement?.className === 'messages-list') ||
-						wheelScrollDirection === 'next')
-				) {
-					loadNextMessages().then(messages => onNextChannelResponse(messages));
-				}
-
-				if (
-					isEndReached &&
-					searchChannelPrev &&
-					!searchChannelPrev.isLoading &&
-					!searchChannelPrev.isEndReached &&
-					!isHorizontal &&
-					((wheelScrollDirection === undefined &&
-						scroller.parentElement?.className === 'messages-list') ||
-						wheelScrollDirection === 'previous')
-				) {
-					loadPrevMessages().then(messages => onPrevChannelResponse(messages));
-				}
+	const debouncedScrollHandler = useDebouncedCallback((event: React.UIEvent<'div'>) => {
+		const scroller = event.target;
+		if (scroller instanceof Element) {
+			const isStartReached = scroller.scrollTop === 0;
+			if (isStartReached && searchChannelNext && !searchChannelNext.isLoading) {
```

Comment:
what if live update is on? besides UpdatesStore making requests you make another one here